packages/scheduler/src/forks/Scheduler.js

```diff
@@ -557,18 +557,19 @@ function requestHostTimeout(
   callback: (currentTime: number) => void,
   ms: number,
 ) {
-  // $FlowFixMe[not-a-function] nullable value
-  taskTimeoutID = localSetTimeout(() => {
+  // Use queueMicrotask instead of setTimeout
+  queueMicrotask(() => {
     callback(getCurrentTime());
-  }, ms);
+  });
```

Comment:
This callsite is trying to schedule a delayed task in the future, not a microtask. So this would break a lot of things.